slick.remotemodel.ts

```diff
@@ -1,3 +1,10 @@
+// TODO: remove when slick.core has been converted
+declare namespace Slick {
+  export class Event {
+    notify(args: any): void;
+  }
+}
```

Comment:
Hack to get it compiling.  Event exists in `slick.core`, but it has not been migrated yet.  Once it has, we should be able to to reference the type from there instead.